### PR TITLE
added event listener to clear todos

### PR DESCRIPTION
**Pull request by JauniusZ**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -65,3 +65,12 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+document
+  .getElementById("clearCheckedTodosBtn")
```
Comment (arturasmckwcz) [inline script.js:70]: NIT
a bit different pattern to defining consts at the top and then use them for setting listeners

File: script.js [hunk 2]
```diff
@@ -65,3 +65,12 @@ function updateUncheckedTodoCount() {
...
+  .addEventListener("click", () => {
+    state.todos = state.todos.filter((todo) => !todo.checked);
```
Comment (arturasmckwcz) [inline script.js]: NIT
you may use de-structuring:
```
state.todos = state.todos.filter(({ checked }) => !checked);
```

File: script.js [hunk 3]
```diff
@@ -65,3 +65,12 @@ function updateUncheckedTodoCount() {
...
+
+    renderTodos();
+    updateUncheckedTodoCount();
```
Comment (arturasmckwcz) [inline script.js]: You do not need to update the count cos this click of the button has nothing to do with check/uncheck of a todo.

**Comment by JauniusZ** [comment]
Made the changes.

**Review by arturasmckwcz — APPROVED** [review]
cool! thank you

**Pull request merged**